recipe/meta.yaml

```diff
@@ -1,43 +1,81 @@
 {% set name = "tiledbsoma" %}
-{% set version = "0.5.0a2" %}
+{% set version = "1.0rc0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e0fb16e55e01315a46f102739dcd3d1b15f24d636e00a2d737c0dde674b86370
-
-build:
-  number: 0
-  script: python -m pip install . -vv
-  skip: true  # [win]
+  git_url: https://github.com/single-cell-data/TileDB-SOMA.git
+  git_rev: 8a57a05edecc2b2e17675dc59791c2dedc764034
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
     - cmake >=3.21
     - make
   host:
-    - pip
+    - tiledb 2.14.*
     - pybind11
-    - python
-    - setuptools
-    - setuptools_scm
-    - wheel
+    - spdlog
   run:
-    - anndata
-    - pandas
-    - pyarrow
-    - python
-    - scanpy
-    - scipy
-    - tiledb-py
+    - tiledb 2.14.*
+    - spdlog
+
+outputs:
+  - name: libtiledbsoma
+    version: {{ version }}
+    script: build-libtiledbsoma.sh
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - {{ compiler('c') }}
+        - cmake
+        - make
+        - pybind11
+      host:
+        - tiledb 2.14.*
+        - spdlog
+      run:
+        - tiledb 2.14.*
+        - spdlog
 
-test:
-  commands:
-    - python -c "import tiledbsoma.libtiledbsoma; print(tiledbsoma.libtiledbsoma.version())"
+  - name: tiledbsoma-py
+    version: {{ version }}
+    script: build-tiledbsoma-py.sh
+    build:
+      binary_relocation: 
+        - lib/python3.10/site-packages/tiledbsoma/libtiledbsoma.*
+        - lib/python3.10/site-packages/tiledbsoma/pytiledbsoma.*.so
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - {{ compiler('c') }}
+        - cmake >=3.21
+        - make
```

Comment:
Are `cmake` and `make` build dependencies for the Python client when it is building against a pre-built libtiledbsoma? As a related example, building tiledbvcf-py doesn't require `cmake` and `make` when it builds against a pre-built libtiledbvcf (https://github.com/TileDB-Inc/tiledb-vcf-feedstock/pull/78)